org.nongnu.enigma.yml

```diff
@@ -0,0 +1,83 @@
+app-id: org.nongnu.enigma
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: enigma
+rename-icon: enigma
+rename-desktop-file: enigma.desktop
+
+finish-args:
+  - --persist=.enigma
+  # X11 access
+  - --share=ipc
+  - --socket=x11
+  # Needs to talk to pulseaudio
+  - --socket=pulseaudio
+  # OpenGL redering
+  - --device=dri
+
+modules:
+  - name: texi2html
+    sources:
+      - type: archive
+        url: https://download.savannah.gnu.org/releases/texi2html/texi2html-5.0.tar.gz
+        sha256: e60edd2a9b8399ca615c6e81e06fa61946ba2f2406c76cd63eb829c91d3a3d7d
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .
+
+  - name: graphviz
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/2.49.0/graphviz-2.49.0.tar.gz
+        sha256: 17ade6189beade7c0784090f283c1f78f6b286ec38e3d2e927d3e42d625d0e8d
+
+  - name: Xerxes-C
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://dlcdn.apache.org//xerces/c/3/sources/xerces-c-3.2.3.tar.gz
+        sha256: fb96fc49b1fb892d1e64e53a6ada8accf6f0e6d30ce0937956ec68d39bd72c7e
+
+  - name: imagemagick
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-modules
+      - --with-x
+      - --with-threads
+      - --with-magick_plus_plus
+      - --with-gslib
+      - --with-wmf
+      - --with-webp
+      - --with-openexr
+      - --with-rsvg
+      - --with-xml
+      - --with-jbig
+      - --with-openjp2
+    sources:
+      - type: git
+        url: https://github.com/ImageMagick/ImageMagick.git
+        tag: "7.1.0-5"
+        commit: 74f8154136d2f631ca3e0421e6cf6c8dccd548f7
+
+  - name: Enigma-Game
+    no-autogen: false
+    sources:
+      - type: git
+        url: https://github.com/MrTarantoga/Enigma.git
+        commit: 0db9df0aea0c7a874a494647210265340425bdd9
+      - type: file
+        path: org.nongnu.enigma.metainfo.xml
+    post-install:
+      - install -d /app/share/icons/hicolor/128x128/apps/
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - convert /app/share/icons/hicolor/48x48/apps/enigma.png -resize 128x128 /app/share/icons/hicolor/128x128/apps/enigma.png
+
+cleanup:
```

Comment:
By convention, a global `cleanup` section is usually above `modules`.

Reply:
Fixed in [commit](https://github.com/flathub/flathub/pull/2653/commits/dccee3916d5c05efa9ecbe2e15b401dc15c10585)